sources/TileDB.Interop/LibDllImport.cs

```diff
@@ -2,16 +2,13 @@
 {
     internal class LibDllImport
     {
-#if __OSX__
-        public const string Path = "libtiledb.dylib";
-        public const string LibCPath = "libc.dylib";
-#elif __LINUX__
-        public const string Path = "libtiledb.so.2.9";
-        public const string LibCPath = "libc.so";
-#else
-        public const string Path = "tiledb.dll";
-        public const string LibCPath = "msvcrt.dll";
-#endif
-   
+        public const string Path = "tiledb";
```

Comment:
How about keeping the conditional `libtilebd.so.2.9` on linux and returning this on windows/mac?

Reply:
I think it will work, but what is the motivation for this? I guess it will make things a little bit easier with CI (only for the release) and the several xml files, but will leave us with the conditional for the OS which Imo is also very frustrating (with no reason as it turns out.) I am good with either case